src/components/swapv2/PairSuggestion/SearchInput.tsx

```diff
@@ -114,8 +114,13 @@ export default forwardRef<HTMLInputElement, Props>(function SearchComponent(
         ) : (
           <InputIcon onClick={showListView} key={2}>
             <Flex>
-              <Command size={13} />
-              <span style={{ marginLeft: 3 }}>K</span>
+              {isMacOs ? (
+                <>
+                  <Command size={13} /> <span style={{ marginLeft: 3 }}>K</span>
+                </>
+              ) : (
+                <span>ctrl+k</span>
```

Comment:
Dí @Linhhnk for logo for Ctrl button :tien-sword:

Reply:
"Ctrl + K" please